apps/web/src/index.css

```
@@ -38,6 +39,28 @@ code,
   outline-offset: 2px;
 }
 
+@layer components {
+  .console-label {
+    @apply text-sm uppercase;
+    letter-spacing: 0;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)